modules/programs/autojump.nix

```diff
@@ -0,0 +1,61 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.autojump;
+in {
+  meta.maintainers = [ maintainers.evanjs ];
+
+  options.programs.autojump = {
+    enable = mkEnableOption "autojump";
+
+    package = mkOption {
```

Comment:
I think you can remove the package option, I don't think it will be particularly common that people would like to manipulate the package. I.e., can use `pkgs.autojump` directly.

Reply:
Sounds good! Had to add it to lorri lately and I'm still trying to get a good feel of when including it is right (if only it was done implicitly...)